core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -399,4 +435,60 @@ public Response getWorkerInfoList() {
       }
     });
   }
+
+  private Capacity getCapacityInternal() {
+    return new Capacity().setTotal(mBlockMaster.getCapacityBytes())
+        .setUsed(mBlockMaster.getUsedBytes());
+  }
+
+  private Map<String, String> getConfigurationInternal() {
+    Set<Map.Entry<String, String>> properties = Configuration.toMap().entrySet();
+    SortedMap<String, String> configuration = new TreeMap<>();
+    for (Map.Entry<String, String> entry : properties) {
+      String key = entry.getKey();
+      if (PropertyKey.isValid(key)) {
+        configuration.put(key, entry.getValue());
+      }
+    }
+    return configuration;
+  }
+
+  private Map<String, Long> getMetricsInternal() {
```

Comment:
Manager frontend needs to distinguish between different types of metrics, e.g. RPC invocation counters, file system operations.